xdebug_gc_stats.h

```diff
@@ -28,7 +28,7 @@ typedef struct _xdebug_gc_run {
 	char        *class_name;
 } xdebug_gc_run;
 
-int (*xdebug_old_gc_collect_cycles)(void);
+extern int (*xdebug_old_gc_collect_cycles)(void);
```

Comment:
This breaks the Travis build: https://travis-ci.org/xdebug/xdebug/jobs/324114059#L774

Reply:
You also need to add ``int (*xdebug_old_gc_collect_cycles)(void);`` to xdebug_gc_stats.c

Reply:
Fixed, thanks.